HealthcareSystem/PatientWebApp/Controllers/PatientController.cs

```diff
@@ -177,13 +177,17 @@ public IActionResult GetMaliciousPatients()
 
 
         [HttpPut("blocked/{jmbg}")]
-        public ActionResult CancelExamination(string jmbg)
+        public ActionResult BlockPatient(string jmbg)
         {
             try
             {
                 _patientService.BlockPatient(jmbg);
                 return Ok();
             }
+            catch (NotFoundException exception)
+            {
+                return NotFound(exception.Message);
```

Comment:
I suppose a discussion could be had on whether NotFound or BadRequest is a more appropriate return code, but either is good enough in this particular case I think.